src/main/java/io/ebeaninternal/server/query/SqlTreeBuilder.java

```diff
@@ -435,7 +435,7 @@ private void addProperty(SqlTreeProperties selectProps, BeanDescriptor<?> desc,
       BeanProperty p = desc.findBeanProperty(propName);
       if (p == null) {
         logger.error("property [" + propName + "] not found on " + desc + " for query - excluding it.");
-        p = desc.findBeanProperty("id");
```

Comment:
If I remember correctly, I ran into an NPE when using query.select("...wrong_spelled_prop...") on an entity which primary key was not named "id" but "key". Should I provide a test case for this?